src/rdo.rs

```diff
@@ -354,6 +356,8 @@ pub fn rdo_mode_decision(
         let wr: &mut dyn Writer = &mut WriterCounter::new();
         let tell = wr.tell_frac();
 
+        if skip { tx_type = TxType::DCT_DCT; };
```

Comment:
Is this a bug fix?

Reply:
heh, oops, midair collision.  Question still stands :-)

Reply:
It IS a FIX! :) Since the tx_type, by def of av1, should be DCT_DCT if skip is true.